### ISSUE TITLE
Default example not work when specific DOCTYPE html

### ISSUE THREAD

**Issue by kelvinkoko** [davidfig/window-manager#11]
I tried the basic example, with a simple HTML (empty body)
It work as expected (Snap to browser edge, set default window x, y)
But if i added `DOCTYPE html`
```
<!DOCTYPE html>
<head>
    <meta charset="utf-8">
    <title>wm Example</title>
    <style>
        body {
            background: #aaaaaa;
        }
    </style>
</head>
<body>
    <script src="main.js"></script>
</body>
```
Snap to browswer edge, and set window position not work.
I think it is related to the body height problem
as state here:
https://stackoverflow.com/questions/1966300/height-100-is-not-working-in-html-when-using-doctype

After adding ` html, body { height: 100% }`, snap to edge work (because body height filled viewport) But the setting window position in createWindow still not work. Do you think of some potential reason? Thank you

**Comment by davidfig** [comment]
Good catch. There was a bug with th the createWindow() code (it was assigning the style.left/top without 'px'). It's fixed in v1.2.0. 

You'll need the html, body: { height: 100% } to keep it fullscreen. 

Let me know if you have any other issues.

**Comment by kelvinkoko** [comment]
Thanks for the prompt reply!~ It work perfectly now!

Btw, this library is great, it is simple enough but do what it is targeted to do very well. Thank you very much!
